vulcanai2/models/metrics.py

```diff
@@ -126,6 +126,186 @@ def extract_class_labels(self, in_matrix):
         elif in_matrix.shape[1] == 1:
             return np.around(in_matrix)
 
+    # TODO: check types
+    def get_confusion_matrix_values(self, confusion_matrix):
```

Comment:
@rfratila what do you think, should this be one function or four?

Reply:
like `get_tp()`, `get_fp(`), etc? you do need some to calculate others so I think it's best to keep these together?